### PR TITLE
fixes #19319 - Error on deleting a product is unclear and has nothing…

### PR DESCRIPTION
**Pull request by sjha4**
… to do with sync plans

### CODE REVIEW

**Comment by sjha4** [comment]
## 19319:
### Error message on deleting a RH product isn't clear
#### Original error message:
"An error occurred updating the sync plan: " + message
#### Updated error message: 
"An error occurred : " + message

**Comment by dLobatog** [comment]
[test]

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.controller.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ angular.module('Bastion.products').controller('ProductsController',
 
         bulkError = function (response) {
             angular.forEach(response.data.errors, function(message) {
-                GlobalNotification.setErrorMessage(translate("An error occurred updating the sync plan: ") + message);
+                GlobalNotification.setErrorMessage(translate("An error occurred : ") + message);
```
Comment (johnpmitsch) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.controller.js]: Nitpick - it really should be `An error occurred:` (no space between colon and occurred)

**Comment by johnpmitsch** [comment]
@sjha4 thanks!

**Comment by johnpmitsch** [comment]
@chris1984 oversight on my part, I clicked the shiny green button without checking the number of commits and telling to @sjha4 to squash into one

**Pull request merged**